compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -2033,6 +2033,19 @@ OMR::SymbolReferenceTable::findOrCreateOSRFearPointHelperSymbolRef()
    return element(osrFearPointHelperSymbol);
    }
 
+TR::SymbolReference *
+OMR::SymbolReferenceTable::findOrCreateEAEscapeHelperSymbolRef()
+   {
+   if (!element(eaEscapeHelperSymbol))
+      {
+      TR::MethodSymbol* sym = TR::MethodSymbol::create(trHeapMemory(), TR_None);
```

Comment:
Is the rationale for TR_None here that it is not a helper in the sense that it is'nt expected to reach the codegen even ?

Reply:
correct